packages/meditrak-server/src/tests/routes/GETDashboardItems.test.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import {
+  findOrCreateDummyRecord,
+  addBaselineTestCountries,
+  buildAndInsertProjectsAndHierarchies,
+} from '@tupaia/database';
+import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, BES_ADMIN_PERMISSION_GROUP } from '../../permissions';
+import { TestableApp, setupDashboardTestData } from '../testUtilities';
+
+describe('Permissions checker for GETDashboardItems', async () => {
+  const DEFAULT_POLICY = {
+    DL: ['Public'],
+    KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+    VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    LA: ['Admin'],
+    TO: ['Admin'],
+  };
+
+  const BES_ADMIN_POLICY = {
+    LA: [BES_ADMIN_PERMISSION_GROUP],
+  };
+
+  const app = new TestableApp();
+  const { models } = app;
+  let districtDashboardItem1;
+  let nationalDashboardItem1;
+  let nationalDashboardItem2;
+  let nationalDashboardItem3;
+  let projectDashboardItem1;
+  let filterString;
+
+  before(async () => {
+    // Still create these existing entities just in case test database for some reasons do not have these records.
+    await findOrCreateDummyRecord(models.entity, {
+      code: 'KI_Phoenix Islands',
+      type: 'district',
+      country_code: 'KI',
+    });
+
+    await addBaselineTestCountries(models);
+
+    await buildAndInsertProjectsAndHierarchies(models, [
+      {
+        code: 'test_project',
+        name: 'Test Project',
+        entities: [{ code: 'KI' }, { code: 'VU' }, { code: 'TO' }, { code: 'SB' }],
+      },
+    ]);
+
+    ({
+      districtDashboardItem1,
+      nationalDashboardItem1,
+      nationalDashboardItem2,
+      nationalDashboardItem3,
+      projectDashboardItem1,
+    } = await setupDashboardTestData(models));
+
+    const dashboardItemIds = [
+      districtDashboardItem1.id,
+      nationalDashboardItem1.id,
+      nationalDashboardItem2.id,
+      nationalDashboardItem3.id,
+      projectDashboardItem1.id,
+    ];
+    filterString = `filter={"id":{"comparator":"in","comparisonValue":["${dashboardItemIds.join(
+      '","',
+    )}"]}}`;
+  });
```

Comment:
You can move this inside the `describe('GET /dashboardItems'` block (line 153), since it is only used there